README.md

```diff
@@ -19,7 +21,34 @@ python3 -m uvicorn fastapi_app:app --reload --port=8000
 
 ```bash
 cd src/frontend
+npm install
 npm run dev
 ```
 
-1. Open the browser at `http://localhost:5173/` and you will see the frontend.
\ No newline at end of file
+1. Open the browser at `http://localhost:5173/` and you will see the frontend.
+
+## Setup permissions for Azure OpenAI endpoint
+
+If the permisions have not yet been setup, you can configure them using the following script:
```

Comment:
I am working on a branch that adds azd, so it would add the roles required, but we can also provide this guidance for developers who dont care about deploying the whole app yet but do want Azure OpenAI locally. What about just adding this code as a script to the repo? We have a similar optional script in azure-search-openai-demo for setting roles. Then we can easily run it to make sure it works.

Reply:
Yeah, that's actually where I copied the code from, that works, I'll add it

Reply:
Added the bash version